azure/resource-group/README.md

```diff
@@ -33,11 +33,13 @@ module "resource-group" {
   source                    = "../tf-modules/azure/resource-group"
   name                      = "rg-my_project-dev"
   location                  = "West Europe"
+  environment = "Dev"
+  country     = "nl"
+  squad       = "infra"
+  product     = "internal"
   tags                      = {
-    country : "nl"
-    squad : "infra"
-    product : "test-product"
-    environment : "prod"
+    datadog : "true"
+    ansible : "true"
```

Comment:
no clue what is this about?

Reply:
It was just an example to show that we could add extra tags, but I guess they were not good examples 😔